lib/foreman_inventory_upload/async/upload_report_job.rb

```diff
@@ -9,22 +9,32 @@ def self.output_label(label)
 
       def perform(filename, organization_id)
         label = UploadReportJob.output_label(organization_id)
+        @filename = filename
+        @organization = Organization.find(organization_id)
+
+        unless @organization.owner_details&.fetch('upstreamConsumer')&.fetch('idCert')
+          logger.info("Skipping organization #{@organization}, no candlepin certificate defined.")
+          progress_output_for(label) do |progress_output|
+            progress_output.write_line("Skipping organization #{@organization}, no candlepin certificate defined.")
```

Comment:
Can you add quotes around the org name?
```suggestion
            progress_output.write_line("Skipping organization: '#{@organization}', no candlepin certificate defined.")
```

![no_cert](https://user-images.githubusercontent.com/26363699/90728043-e5831400-e2cc-11ea-84e4-24f5e213fd56.png)